packages/histoire-controls/src/components/radio/HstRadio.vue

```diff
@@ -0,0 +1,107 @@
+<script lang="ts">
+export default {
+  name: 'HstRadio',
+}
+</script>
+
+<script lang="ts" setup>
+import { computed, ComputedRef, ref } from 'vue'
+import HstWrapper from '../HstWrapper.vue'
+
+export interface RadioOption {
+  label: string
+  value: string
+}
+
+const props = defineProps<{
+  title?: string
+  modelValue: string
+  options: RadioOption[]
```

Comment:
Wouldn't it be more consistent to have the same type as the Select component 🤔 ?
https://github.com/histoire-dev/histoire/blob/2a64cb52d816b337415211fb29f2c3cc1943776e/packages/histoire-controls/src/components/select/HstSelect.vue#L17

Reply:
Totally agree with you, I was hesitating if adding it or not, where could we add types and interfaces that are global? a `src/types.ts` file?

Reply:
`src/types.ts` looks fine to me 👍 !

Reply:
Done :D